docs/conf/portland/2024/news/welcome.rst

```diff
@@ -0,0 +1,70 @@
+:template: {{year}}/generic.html
+
+.. post:: Oct 12, 2023
+   :tags: {{shortcode}}-{{year}}
+
+Announcing Write the Docs {{ city }} {{year}}
+===========================================
+
+Welcome to the official launch of `Write the Docs Portland {{year}} <https://www.writethedocs.org/conf/{{ shortcode }}/{{ year }}/>`_.
+We're excited to announce the dates for our next conference:
+**Write the Docs {{ city }} {{ year }} will be on {{ date.short }}**
+
+We are excited to welcome everyone back to Revolution Hall in Portland for another year of great talks and community building.
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+**We plan to launch our CFP in November** for those who are excited to speak.
+We're still working out the exact details for the CFP,
+but our current plan is to record all talks so that the experience will be comparable between online and in-person,
+for attendees and speakers.
+We hope that this will help maintain the accessibility level that remote presentations gave us,
+and invite speakers who cannot or do not feel comfortable attending in person.
+
+Start thinking about your talk proposal now,
+and we'll let you know as soon as the CFP opens.
+
+
+Tickets
+-------
+
+**Tickets will go on sale in {{ date.tickets_live }}**.
```

Comment:
I remember every year there are always some people who need to use their travel budget before the year is over, so maybe if we open CFP in November we could open ticket sales in December?

Reply:
Yea, seems reasonable? Overall it's a bit annoying to open tickets in the prior year, but not a huge issue.